src/cf-vars.less

```diff
@@ -44,15 +47,18 @@
     - cf-core
 */
 
-// Sizing variables 
+// Sizing variables
 
 @base-font-size-px:             16px;
 @base-line-height-px:           22px;
 @base-line-height:              unit(@base-line-height-px / @base-font-size-px);
 @mobile-max:                    599px;
 @tablet-min:                    600px;
+@tablet-max:                    800px;
+@desktop-min:                   801px;
+@wall-min:                      1200px;
```

Comment:
@anselmbradford is correct, devices typically on a wall, as in TVs. It has nothing to do with the edges of the layout because the layout could conceivably work for larger screen devices without the "pillboxing" that currently happens.